examples/MomentoApplication/Program.cs

```diff
@@ -8,97 +8,139 @@
-            break;
+            // Also not considered fatal.
+            Console.WriteLine($"Error getting value: {getError.Message}!");
         }
-    } while (!String.IsNullOrEmpty(token));
 
-    Console.WriteLine($"\nSetting key: {KEY} with value: {VALUE}");
-    var setResponse = await client.SetAsync(CACHE_NAME, KEY, VALUE);
-    if (setResponse is CacheSetResponse.Error setError)
-    {
-        // Warn the user of the error and exit.
-        Console.WriteLine($"Error setting value: {setError.Message}. Exiting.");
-        Environment.Exit(1);
+        Console.WriteLine($"\nDeleting key {KEY}");
+        var deleteKeyResponse = await client.DeleteAsync(CACHE_NAME, KEY);
+        if (deleteKeyResponse is CacheDeleteResponse.Error deleteKeyError)
+        {
+            // Also not considred fatal.
+            Console.WriteLine($"Error deleting key: {deleteKeyError.Message}!");
+        }
     }
 
-    Console.WriteLine($"\nGetting value for key: {KEY}");
-    CacheGetResponse getResponse = await client.GetAsync(CACHE_NAME, KEY);
-    if (getResponse is CacheGetResponse.Hit getHit)
-    {
-        Console.WriteLine($"Looked up value: {getHit.ValueString}, Stored value: {VALUE}");
-    }
-    else if (getResponse is CacheGetResponse.Miss)
-    {
-        // This shouldn't be fatal but should be reported.
-        Console.WriteLine($"Error: got a cache miss for {KEY}!");
-    }
-    else if (getResponse is CacheGetResponse.Error getError)
-    {
-        // Also not considered fatal.
-        Console.WriteLine($"Error getting value: {getError.Message}!");
-    }
+    public static async Task DeleteCacheExample(ISimpleCacheClient client) {
 
-    Console.WriteLine($"\nDeleting key {KEY}");
-    var deleteKeyResponse = await client.DeleteAsync(CACHE_NAME, KEY);
-    if (deleteKeyResponse is CacheDeleteResponse.Error deleteKeyError)
-    {
-        // Also not considred fatal.
-        Console.WriteLine($"Error deleting key: {deleteKeyError.Message}!");
+        Console.WriteLine($"\nDeleting cache {CACHE_NAME}");
+        var deleteCacheResponse = await client.DeleteCacheAsync(CACHE_NAME);
+        if (deleteCacheResponse is DeleteCacheResponse.Error deleteCacheError)
+        {
+            // Report fatal error and exit
+            Console.WriteLine("Error deleting cache: {deleteCacheError.Message}. Exiting.");
+            Environment.Exit(1);
+        }
     }
 
-    Console.WriteLine($"\nDeleting cache {CACHE_NAME}");
-    var deleteCacheResponse = await client.DeleteCacheAsync(CACHE_NAME);
-    if (deleteCacheResponse is DeleteCacheResponse.Error deleteCacheError)
+    /// <summary>
+    /// By default, cache clients will connect lazily when the first request is
+    /// issued.  This example shows how to configure a client to connect eagerly
+    /// when it is instantiated, and to specify a distinct timeout for the
+    /// connection to be established.
+    /// </summary>
+    public static async Task EagerConnectionExample()
```

Comment:
The GitHub checks complain at this line saying there are no awaited expressions below, so the function should not be async. Does the constructor await the eager connection call?

Reply:
Is there a pre-completed task that can be returned like completedFuture in Java?

Reply:
the constructor call results in a synchronous call to force the connection, so you're right, I don't need async here.

Reply:
addressed in https://github.com/momentohq/client-sdk-dotnet/pull/363/commits/299c9ad80feb332b48219f4d4afc350ffc436bf9